x/machine/keeper/issue_response.go

```diff
@@ -0,0 +1,25 @@
+package keeper
+
+import (
+	"github.com/cometbft/cometbft/libs/log"
+	"github.com/hypebeast/go-osc/osc"
+)
+
+func (k Keeper) IssueResponseHandler(logger log.Logger) {
+	addr := "0.0.0.0:8766"
```

Comment:
Is it a good idea to hard-code the port like that?

Reply:
it's not a good idea. but it's in any case temporarily. will change it.